src/server/db/schema.ts

```diff
@@ -112,8 +112,9 @@ export const verificationTokens = mysqlTable(
 export const videoEntries = mysqlTable("videoEntry", {
   id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
   uploadId: varchar("uploadId", { length: 256 }).notNull(),
-  assetId: varchar("assetId", { length: 256 }),
+  assetId: varchar("assetId", { length: 255 }),
```

Comment:
Why you've changed length from `256` to `255`?

Reply:
Length of `assetId` is capped by mux to `255`.  Can't copy link to highlight for some reason: https://docs.mux.com/webhook-reference#video.asset.ready